src/components/App.js

```diff
@@ -1,8 +1,196 @@
+import React, { useState } from 'react';
+import SelectNumber from './Select';
+import Board from './Board';
+import { alphabet } from '../letters/Alphabet';
+import { wordDictionary2 } from '../letters/WordDictionary2';
+import { wordDictionary3 } from '../letters/WordDictionary3';
+import { wordDictionary4 } from '../letters/WordDictionary4';
+import { wordDictionary5 } from '../letters/WordDictionary5';
+import { letterPoints } from '../letters/LetterPoints';
 
 function App() {
+  const [selectNumber, setSelectNumber] = useState(false);
+  const [wordNo, setWordNo] = useState(1);
+  const [numberSelected, setNumberSelected] = useState('');
+  const [squares, setSquares] = useState([]);
+  const [wordLengths, setWordLengths] = useState([]);
+  const [remainingAlphabet, setRemainingAlphabet] = useState(alphabet);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [score, setScore] = useState(0);
+  let game;
+  let words = [];
+  let invalidWord = '';
+
+  class Game {
+    restart() {
+      setSelectNumber(false);
+      setNumberSelected('');
+      setWordLengths([]);
+      setSquares([]);
+      setWordNo(1);
+      setErrorMessage('');
+      setRemainingAlphabet(alphabet);
+      setScore(0);
+      console.clear();
+    }
+    calculateScore(words) {
+      let workScore = 20 - numberSelected;
+      for (let i = 0; i < words.length; i++) {
+        for (let j = 0; j < words[i].length; j++) {
+          workScore =
+            workScore +
+            letterPoints.find(item => {
+              return item.letter === words[i][j].toUpperCase();
+            }).point;
+      }
+    }
+    return workScore;
+    }
+  };
+  game = new Game();
+
+  const handleClick = () => {
```

Comment:
This should have a better name like `onDoneClicked`

Reply:
Done.